[Alerting][Docs] Adding template for documenting alert and action types (#92830)
elastic/kibana: docs/user/alerting/defining-alerts.asciidoc

{kib} alerts can be created in a variety of apps including <<xpack-apm,*APM*>>, <<metrics-app,*Metrics*>>, <<xpack-siem,*Security*>>, <<uptime-app,*Uptime*>> and from <<management,*Management*>> UI. While alerting details may differ from app to app, they share a common interface for defining and configuring alerts that this section describes in more detail.

[float]

> === Creating an alert

Reviewer: ```suggestion
=== Create an alert
```

Also change "Managing alerts" to "Manage alerts"

> === Create an alert

Reviewer: All alerts share the following four properties in common: >

All alerts share the following four properties.

> When you create an alert, you need to define the alert details, the conditions and the actions.

Reviewer: ```suggestion
When you create an alert, you must define the alert details, conditions, and actions.
```
